Misc/NEWS.d/next/Library/2020-05-14-16-01-34.bpo-40592.Cmk855.rst

```diff
@@ -0,0 +1 @@
+shutil.which returns not None anymore with ; at the end of PATHEXT
```

Comment:
Proposal:
```suggestion
:func:`shutil.which` now ignores empty entries in :envvar:`PATHEXT` instead of treating them as a match.
```

(I hope I got the escapes right - we should check the doc build output for warnings)